### DISCUSSION TITLE
Construct `ORDER_BY` statement by itself

### DISCUSSION THREAD

**Discussion by Zizico2** [SeaQL/sea-query#586] [Q&A]
Is it possible to create an `ORDER_BY` statement by itself? There are tools outside of DBs that use the sql syntax for ordering. It would be nice to be able to generate those with sea query. For now I can generate a full select and then just remove what I don´t want.

**Comment by tyt2y3** [comment]
That's good enough right. Provided you have a very simple `SELECT * FROM table` and replace it with empty string.

> There are tools outside of DBs that use the sql syntax for ordering

Although I am curious what tools are you referring? What is the use case?

**Comment by Zizico2** [comment]
For my use case:

any `gRPC` server that follows these design patterns [https://cloud.google.com/apis/design/design_patterns](https://cloud.google.com/apis/design/design_patterns). Including some internal ones, made by my team. We've been using that as a guideline, and they recommend having the `order_by` field be a SQL-like string, which we've been doing. Having something to write the SQL would be great.
